src/recipe/parser/build.rs

```diff
@@ -55,9 +55,17 @@ pub struct Build {
     /// Setting to control wether to always include a file (even if it is already present in the host env)
     #[serde(default, skip_serializing_if = "GlobVec::is_empty")]
     pub(super) always_include_files: GlobVec,
+    /// Merge the build and host envs
+    #[serde(default)]
```

Comment:
Would be cool to skip rendering this if it's false (the default).

I think I added a `is_false` function somewhere. We could move it to a `serde_utils.rs` file.

Reply:
We can use `std::ops::Not::not`